gammapy/maps/region.py

```diff
@@ -170,6 +218,14 @@ def crop(self):
         raise NotImplementedError("Cropping of `RegionGeom` not supported")
 
     def solid_angle(self):
+        """Get solid angle of the region.
```

Comment:
Here is some Gammapy developing weirdness: for `Map` objects the docstrings are inherited from the base class. If you check https://github.com/gammapy/gammapy/blob/master/gammapy/maps/geom.py#L2199, you can see, that the doctoring is already implemented. The problem is how to find whether it's missing? I think the best way for now is just to check the docstring interactively using `help(geom.solid_angle)`...if you think the docstring can be  improved, you can either edit th one in the base class, or re-implement it for the `RegionGeom` (where sometimes return types might differ...)

Reply:
Oh right, I had noticed this "weirdness" too, but somehow missed that `solid_angle` already had a docstring. I already modified some that, even though they had the base class docstring, the text didn't quite describe the `RegionGeom` implementation of the method (like in the case in your comment below). I will be more careful and consistent with this though, thanks!

Reply:
As long as the docstring you have now is more complete than the one in the base class, just keep it. For developers it can be also nice to have the docstring close to the implementation of the method.